config/protocol.privnet.docker.four.yml

```diff
@@ -43,6 +43,17 @@ ApplicationConfiguration:
   MaxPeers: 10
   AttemptConnPeers: 5
   MinPeers: 3
+  Oracle:
+    Enabled: true
```

Comment:
I wouldn't be so sure to enable it by default for privnet installations.

Reply:
Fixed.